redis/reply.go

```diff
@@ -345,6 +345,24 @@ func Int64s(reply interface{}, err error) ([]int64, error) {
 	return result, err
 }
 
+func Uint64s(reply interface{}, err error) ([]uint64, error) {
+	var result []uint64
+	err = sliceHelper(reply, err, "Uint64s", func(n int) { result = make([]uint64, n) }, func(i int, v interface{}) error {
+		switch v := v.(type) {
+		case uint64:
```

Comment:
This case will not be executed.  Add int64 case and return error on negative number.